libs/sdk/prettier-schematics/src/schematics/ng-add/ng-add.schematic.spec.ts

```diff
@@ -6,35 +6,49 @@ import {
 import commentJson from 'comment-json';
 import path from 'path';
 
-import { createTestLibrary } from '../testing/scaffold';
+import { createTestApp, createTestLibrary } from '../testing/scaffold';
```

Comment:
I found a bug in the `prettier-schematics` library in how it handles our library workspaces.